packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/EditView/Header/index.js

```diff
@@ -0,0 +1,303 @@
+import React, { memo, useCallback, useMemo, useRef, useState } from 'react';
```

Comment:
diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files